First task to development branch

AndriiKolomiiets opened the pull request.

Commit:
Component development. Page template and topnav element created.

Commit:
Component development.
The task performed in a aem-training directory.

yktaleb commented on core/pom.xml:
```diff
@@ -77,11 +78,26 @@
         </dependency>
+        <dependency>
+            <groupId>com.adobe.aem</groupId>
+            <artifactId>uber-jar</artifactId>
```
It is enough to have this dependency in main pom file

AndriiKolomiiets replied:
When I remove this dependency all Sling imports goes red. I suppose it is necessary to include it in this pom too.

yktaleb commented on core/pom.xml:
```diff
@@ -77,11 +78,26 @@
+
+        <dependency>
+            <groupId>org.apache.geronimo.specs</groupId>
+            <artifactId>geronimo-atinject_1.0_spec</artifactId>
```
Why did you add this dependency?

AndriiKolomiiets replied:
I didn't, it was installed automatically when I created project in maven.
I removed it.

yktaleb commented on core/src/test/java/trainingproject/core/models/TestHelloWorldModel.java:
```diff
@@ -51,7 +51,7 @@ public void setup() throws Exception {
     @Test
     public void testGetMessage() throws Exception {
-        // some very basic junit tests
+        // some very basic junit aaaaa
```
Pull request should contain only useful changes regarding new feature(task)

AndriiKolomiiets replied:
Got it, removed comments.

yktaleb commented on it.launcher/pom.xml:
```diff
@@ -40,7 +40,7 @@
 
     <!-- 
-        To keep the instance under test running and run individual tests
+        To keep the instance under test running and run individual aaaaa
```
The same

AndriiKolomiiets replied:
Removed.

yktaleb commented on ui.apps/src/main/content/META-INF/vault/filter.xml:
```diff
@@ -1,5 +1,6 @@
     <filter root="/apps/training"/>
-    <filter root="/apps/sling" />
+    <filter root="/apps/aem-training"/>
+    <filter root="/apps/sling"/>
```
why did you leave <filter root="/apps/sling"/> and  <filter root="/apps/training"/> ?

AndriiKolomiiets replied:
Removed all excessive paths, left just /apps/training

yktaleb commented on ui.apps/src/main/content/jcr_root/apps/aem-training/components/topnav/TopNav.java:
```diff
@@ -0,0 +1,31 @@
+import com.day.cq.wcm.api.Page;
+import com.day.cq.wcm.api.PageFilter;
+
+    public class TopNav extends WCMUsePojo {
```
ctrl + shift + alt + L in order to reformat file :)

yktaleb reviewed (commented):
1. Let's remove all packages/files which are not needed under AEM-training-project\ui.apps\src\main\content\jcr_root\apps.
2. Do not see template

Commit:
Component development.
-Page-component superType corrected.
-"topnav" changed in order to list inner pages. Page, even inner, are listed in one line.

Commit:
Component development.
Changed "topnav" in order to display all inner page links.

Commit:
Component development.
Mistakes were corrected.
